CMakeLists.txt

```diff
@@ -276,7 +285,7 @@ endif()
 #
 find_program(
   CLANG_FORMAT_EXECUTABLE
-  NAMES clang-format clang-format-3.9 clang-format-3.8
+  NAMES clang-format-3.8
```

Comment:
That makes sense. I believe LLVM has a policy of biannual releases with *no guarantee* of backwards compatibility. So it's not surprising that `clang-format` changed significantly between versions. Can you just add a comment to this effect?